routing/graph.go

```diff
@@ -33,6 +33,59 @@ type ReadOnlyGraph interface {
 	Graph
 }
 
+type GraphSessionConstructor interface {
```

Comment:
Above, we still return `kvbd.RTx`, which _assumes_ the backend re-uses the `kvbd` abstractions, which may not necessarily be the case.